ext/php7/engine_api.c

```diff
@@ -121,10 +127,11 @@ ZEND_RESULT_CODE ddtrace_call_function(zend_function **fn_proxy, const char *nam
     // ZVAL_COPY_VALUE(&fci.function_name, fcc.function_handler->common.function_name);
 
     fci.retval = retval;
-    fci.params = argv;
     fci.no_separation = 0;  // allow for by-ref args
-    fci.param_count = argc;
     ZEND_RESULT_CODE result = zend_call_function(&fci, &fcc);
+
+    zend_fcall_info_args_clear(&fci, 1);
```

Comment:
Can you explain why this moved?

Reply:
This was added because `zend_fcall_info_argv()` on line 92 will alloc `fci.params` and copy all the zvals instead of relying on the caller to do that.